searchlib/src/vespa/searchlib/attribute/multinumericattribute.hpp

```diff
@@ -183,4 +184,28 @@ MultiValueNumericAttribute<B, M>::onInitSave(vespalib::stringref fileName)
         (std::move(guard), this->createAttributeHeader(fileName), this->_mvMapping);
 }
 
+template <typename B, typename M>
+long
+MultiValueNumericAttribute<B, M>::onSerializeForAscendingSort(DocId doc, void* serTo, long available, const common::BlobConverter*) const
+{
+    attribute::NumericSortBlobWriter<T, true> writer;
```

Comment:
Consider adding a helper function so lines 191-196 can be shared between this function and _onSerializeForDescendingSort()_.

Reply:
Done.